docs/details/shards/GFX/Viewport.md

```diff
@@ -0,0 +1,5 @@
+- The elements of the int4 output of this shard represent the viewport dimensions in the following format:
```

Comment:
You could add that the origin is at the top-left corner of the window (at 0, 0)
and that it extends positively to the right (+X)
and down (+Y)

Reply:
Also that coordinates are in physical pixels